OMCompiler/SimulationRuntime/c/util/simulation_options.c

```diff
@@ -56,6 +56,7 @@ const char *FLAG_NAME[FLAG_MAX+1] = {
   /* FLAG_HOMOTOPY_MAX_TRIES */           "homMaxTries",
   /* FLAG_HOMOTOPY_NEG_START_DIR */       "homNegStartDir",
   /* FLAG_HOMOTOPY_ON_FIRST_TRY */        "homotopyOnFirstTry",
+  /* FLAG_NO_HOMOTOPY_ON_FIRST_TRY */     "noHomotopyOnFirstTry",
```

Comment:
These two flags (`homotopyOnFirstTry` and `noHomotopyOnFirstTry`) confuse me. Are they somehow connected? How should one use (and possibly combine) them?

Reply:
There are three possibilities:
1. Default (no flag specified): If the model contains homotpy use homotpy method for initialization, otherwise first try it without.
2. homotopyOnFirstTry: Directly use homotpy method for initialization. 
3. noHomotopyOnFirstTry: Don't use homotpy method on first try, even if the model contains the homotpy operator. 

Probably I should make it into on flag that accepts some value. 
But after Christmas holidays.